articles/tokens/id_token.md

```diff
@@ -81,24 +81,28 @@ Once issued, tokens can not be revoked in the same fashion as cookies with sessi
 
 ## Uses
 
-The `id_token` is designed to be used to pass information about a user between websites,  web programs and APIs in an industry standard, URL-friendly fashion.  One advantage of using an `id_token` for this purpose is that the recipient can validate the token without having to make a call back to the issuer of the token.  The token is also designed to enable being passed from one web property to another, via an untrusted client, such that the client cannot alter the token without such tampering being evident to the recipient.
-
 The `id_token` can be used to call the /tokeninfo endpoint within the Auth0 authentication API to retrieve the user’s complete profile.  See the [/tokeninfo endpoint documentation](/auth-api#user-profile) for more details.
 
 The `id_token` can also be used to call the /delegation endpoint within the Auth0 authentication API to obtain another token for another API.  See the [/delegation endpoint documentation](/auth-api#delegated) for more information.
 
-The `id_token` can also be used to call other APIs.
-
 The [Delegation token request sample](https://github.com/auth0/auth0.js#delegation-token-request) provides further examples of using the id_token for other APIs.
 
 This [Animated Sequence Diagram](/sequence-diagrams) shows the sequence of calls used to get an `id_token`.
 
+::: warning-banner
+It is not suggested that you use an `id_token` to secure an API. For this purpose, you should rather make use of [access tokens](/tokens/access_token).
+:::
+
+?? What is the uses of the id token going forward? Access token will now be used in many instances we used id token before ??
```

Comment:
The id token basically has no real use, in the general case.  At best, it is an optimization to avoid network traffic to the /userinfo endpoint.

Consider it this way: the biggest IdP in the world (Facebook) does not use an id_token.  90% of all use cases do not need it.  Keeping that ratio: the access_token should recieve 90% of our attention wrt documenation, and the id_token should recieve 10%.  This is a big shift in mindset.